docs/getting-started/tutorial.md

```diff
@@ -0,0 +1,1001 @@
+- [`__()`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-i18n/) is used for the internationalization of text strings
+
+<div class="callout callout-info">
+	Review the [block wrapper](https://developer.wordpress.org/block-editor/getting-started/fundamentals/block-wrapper/) documentation for an introductory guide on how to ensure the block's markup wrapper has the proper attributes.
+</div>
+
+As a reminder, the main purpose of this block is to display the copyright symbol (©) and the current year. So, you first need to get the current year in string form, which can be done with the following code.
+
+```js
+const currentYear = new Date().getFullYear().toString();
+```
+
+Next, update the function to display the correct information.
+
+```js
+export default function Edit() {
+	const currentYear = new Date().getFullYear().toString();
+
+	return (
+		<p { ...useBlockProps() }>© { currentYear }</p>
+	);
+}
+```
+
+Save the `edit.js` file and refresh the Editor. You will now see the copyright symbol (©) followed by the current year.
+
+![The block in the Editor displays the correct content](https://developer.wordpress.org/files/2023/12/block-tutorial-8.png)
+
+### Updating render.php
+
+While the block is working properly in the Editor, the default block message is still being displayed on the front end. To fix this, open the [`render.php`](https://developer.wordpress.org/block-editor/getting-started/fundamentals/file-structure-of-a-block/#render-php) file, and you will see the following.
+
+```php
+<?php
+...
+?>
+<p <?php echo get_block_wrapper_attributes(); ?>>
+	<?php esc_html_e( 'Copyright Date Block – hello from a dynamic block!', 'copyright-date-block' ); ?>
+</p>
+
+```
+
+Similar to the `useBlockProps()` function in the Editor, [`get_block_wrapper_attributes()`](https://developer.wordpress.org/reference/functions/get_block_wrapper_attributes/) outputs all the necessary CSS classes and styles in the [block's wrapper](https://developer.wordpress.org/block-editor/getting-started/fundamentals/block-wrapper/#the-server-side-render-markup). Only the content needs to be updated.
+
+You can use `date( "Y" )` to get the current year in PHP, and your `render.php` should look like this.
+
+```php
+<?php
+...
+?>
+<p <?php echo get_block_wrapper_attributes(); ?>>© <?php echo date( "Y" ); ?></p>
+```
+
+Save the file and confirm that the block is appearing correctly in the Editor and on the front end.
+
+### Cleaning up
+
+When you use the `create-block` package to scaffold a block, it might include files that you don't need. In the case of this tutorial, the block doesn't use stylesheets or font end JavaScipt. You can clean up the plugin's `src/` folder with the following actions.
+
+1. In the `edit.js` file, remove lines 16-22 that import `editor.scss`
```

Comment:
I would avoid any mentions to specific lines, maybe something like the following is enough
>  In the `index.js` file, remove lines that import `style.scss` and related comments